aptos-move/framework/move-stdlib/tests/vector_tests.move

```diff
@@ -612,20 +615,20 @@ module std::vector_tests {
 
         test_natives_with_type<vector<u8>>(V::empty(), V::empty());
 
-        test_natives_with_type<Droppable>(Droppable{}, Droppable{});
+        test_natives_with_type<Droppable>(Droppable {}, Droppable {});
         (NotDroppable {}, NotDroppable {}) = test_natives_with_type<NotDroppable>(
-            NotDroppable {},
-            NotDroppable {}
+            NotDroppable {}, NotDroppable {}
         );
     }
 
     #[test]
     fun test_for_each() {
         let v = vector[1, 2, 3];
         let s = 0;
-        V::for_each(v, |e| {
-            s = s + e;
-        });
+        V::for_each(v,
+            |e| {
+                s = s + e;
+            });
```

Comment:
This could be considered a bug, or it might not be, because it conflicts with the rules of issue#10[ https://github.com/movebit/movefmt/issues/10]. The current behavior implies that the parameters for the for_each function are in single-line mode, so there is no immediate line break after '(', which affects the line breaking of ')'. We will later try to find alternative ways to optimize the line breaking of ')' to make it as aesthetically pleasing as possible.